templates/zerver/app/lightbox_overlay.html

```diff
@@ -1,4 +1,4 @@
-<div id="lightbox_overlay" class="overlay new-style" data-overlay="lightbox">
+<div role="dialog" aria-labelledby="lightboxTitle"  id="lightbox_overlay" class="overlay new-style" data-overlay="lightbox">
     <div class="image-info-wrapper">
         <div class="image-description">
             <div class="title"></div>
```

Comment:
`aria-labelledby` must reference an existing `id`. I think for the lightbox overlay (which is the overlay shown when clicking on an image), we should add `id="lightboxTitle"` to the above div, which contains the filename.

Reply:
I can't find any div with a filename do you mean this  `<div class="title"></div>`

Reply:
Yes that's what I mean. The div is empty because the content is set dynamically with JavaScript.